docs/moderator-handbook.md

```diff
@@ -0,0 +1,57 @@
+# Jupyter Accessibility Workshop Moderator Handbook!
+
+Hooray! If you are here, you might help moderate one of the upcoming (virtual) Jupyter accessibility workshops. The rest of the organizing team is grateful for your time and care in helping us maintain a welcoming community.
+
+You can find more information about the events on the [Jupyter accessibility workshop GitHub repo](https://github.com/Quansight-Labs/jupyter-accessibility-workshops).
+
+The events will be hosted on [Hopin](https://hopin.com/), which runs on [StreamYard](https://streamyard.com/). You won't need to handle all of these platforms; this is just to let you know where other organizing staff might be in case you need to find them at the event.
+
+## What is a moderator?
+
+
+
+## What can moderators do?
+
+Moderators are responsible for helping event organizers, especially those with admin permissions, maintain an experience that follows the [event code of conduct](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/code-of-conduct.md). In most cases, this includes:
+- Monitoring event chat, video, and audio streams
+- Being present in one breakout room
+- Contacting event organizers in the case of tech issues that can't be resolved
+- Warning participants of Code of Conduct violations if they do not warrant immediate action
+- Contacting event organizers to take removal actions against Code of Conduct violations
+- Taking note of Code of Conduct violations so that they may be reported as a full [Jupyter Code of Conduct violation report]()
+
+For these events, moderators do not have admin permissions. They need to reach out to an admin to take action beyond a warning.
+
+While helping moderate, moderators are free to enjoy the event!
```

Comment:
I think what you're trying to do here is to draw the distinction that moderating is not just sitting on the sidelines and playing referee, but that moderators are encouraged to also participate and have fun in the event. In other words, a moderator is also a participant/attendee, albeit one with a few more responsibilities